packages/elements/src/chart/elements/chart.ts

```diff
@@ -333,8 +327,30 @@ export class Chart extends BasicElement {
     // set global config again in case the CSS font isn't loaded when updating the chart
     this.setGlobalConfig();
     this.decorateColors(chart);
+    this.decorateGridColors(chart);
   };
 
+  /**
+   * Set grid color in beforeUpdate event from Chartjs
+   * @param chart Chart.js instance
+   * @returns {void}
+   */
+  private decorateGridColors(chart: ChartJS): void {
+    for (const scale of Object.keys(chart.scales)) {
+      const axis = chart.options?.scales?.[scale] as CartesianScaleOptions | undefined;
+      const userAxis = this.config?.options?.scales?.[scale] as CartesianScaleOptions | undefined;
+      const userGridColor = userAxis?.grid?.color;
+      const userZeroGridColor = userAxis?.border?.color;
```

Comment:
Zero grid color is no where to be found.
```suggestion
      const userBorderColor = userAxis?.border?.color;
```